lib/charms/traefik_k8s/v0/ingress_per_unit.py

```diff
@@ -89,65 +86,161 @@ def _handle_ingress_per_unit(self, event):
 
 log = logging.getLogger(__name__)
 
-INGRESS_SCHEMA = {
-    "v1": {
-        "requires": {
-            "unit": {
-                "type": "object",
-                "properties": {
-                    "model": {"type": "string"},
-                    "name": {"type": "string"},
-                    "host": {"type": "string"},
-                    "port": {"type": "integer"},
-                },
-                "required": ["model", "name", "host", "port"],
-            }
-        },
-        "provides": {
-            "app": {
-                "type": "object",
-                "properties": {
-                    "ingress": {
-                        "type": "object",
-                        "patternProperties": {
-                            "": {
-                                "type": "object",
-                                "properties": {"url": {"type": "string"}},
-                                "required": ["url"],
-                            }
-                        },
-                    }
-                },
-                "required": ["ingress"],
-            }
-        },
-    }
+# ======================= #
+#      LIBRARY GLOBS      #
+# ======================= #
+
+INTERFACE = "ingress_per_unit"
+ENDPOINT = INTERFACE.replace("_", "-")
```

Comment:
Interface is interface. Endpoint here is "Relation name". @PietroPasotti we should use `DEFAULT_RELATION_NAME` as in https://github.com/canonical/prometheus-k8s-operator/blob/a369c99cfdafeceaf94b1ec3ba1d961a11bb0cc6/lib/charms/prometheus_k8s/v0/prometheus_remote_write.py#L42

Reply:
Umm but in this case I think the usage is appropriate. 
![image](https://user-images.githubusercontent.com/6230162/161697880-078967f6-07ab-4674-9804-c27079768283.png)

In this case 'ingress_per_unit' is the interface, and 'ingress-per-unit' is... what would you call that otherwise?

Reply:
`ingress-per-unit` would be the relation name. `ingress_per_unit` is the relation interface.